BEIMA.Backend.FT/DeviceFT.cs

```diff
@@ -13,7 +13,7 @@ public class DeviceFT : FunctionalTestBase
         public void InvalidId_DeviceGet_ReturnsInvalidId(string id)
         {
             var ex = Assert.ThrowsAsync<HttpRequestException>(async () =>
-                await TestClient.SendRequest("api/device", HttpVerb.GET, queryString: $"id={id}")
+                await TestClient.SendRequest($"api/device/${id}", HttpVerb.GET)
```

Comment:
```suggestion
                await TestClient.SendRequest($"api/device/{id}", HttpVerb.GET)
```
Need to remove '$' from url string.